conf/ledgersmb-nginx.template

```diff
@@ -0,0 +1,63 @@
+##
+# LedgerSMB server configuration
+#
+server {
+        listen 80 default_server;
+        listen [::]:80 default_server;
+
+        #Set to LSMB installed directory
+        root [ROOT];
+
+        index login.pl;
+
+        #Adjust to your liking
+        #server_name ubuntu-vm;
+
+	#Try the asked file directly, pass to Plack if not found
+        location / {
+            try_files $uri @plack;
+        }
+
+        location @plack {
+            proxy_set_header Host $http_host;
+            proxy_set_header X-Forwarded-Host $http_host;
+            proxy_set_header X-Real-IP $remote_addr;
+            proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
```

Comment:
Nice headers. Is it possible to add these to Apache and lighthttpd too (or are they automatic there?)